config/maintainers.json

```diff
@@ -70,6 +70,16 @@
       "link_url": "https://www.sborza.com/",
       "avatar_url": null,
       "bio": "I'm a Canadian (based in Chicago) golang, C and python developer working on projects within the gaming, finance and data-science worlds respectively. My favorite freetime task is coding challenges and I'm more than happy to help people talk through solutions!"
+    },
+    {
+      "github_username": "paparomeo",
+      "alumnus": false,
+      "show_on_website": false,
+      "name": "Pedro Romano",
+      "link_text": "My (inactive) blog",
+      "line_url": "https://paparomeo.dev/",
```

Comment:
```suggestion
      "link_url": "https://paparomeo.dev/",
```

Reply:
Oops, silly me! :disappointed: Rebased and fixed and should be good to go now @cmccandless